ci/pinned: update

wolfgangwalther opened the pull request:
This gives us a [bugfix in treefmt-nix for biome](https://github.com/numtide/treefmt-nix/commit/fb030205684fc4b6dee6c7c6bbcd2bc2523270c5).

From the nixpkgs-unstable channel:
https://hydra.nixos.org/build/312625570#tabs-buildinputs

Changes for treefmt-nix:
https://github.com/numtide/treefmt-nix/compare/f56b1934f5f8fcab8deb5d38d42fd692632b47c2...4ef3dfdbb5ddfb9e39999a2f2b0c2637277859d4

## Things done

- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

wolfgangwalther committed:
ci/pinned: update

This gives us a bugfix in treefmt-nix for biome.

From the nixpkgs-unstable channel:
https://hydra.nixos.org/build/312625570#tabs-buildinputs

Changes for treefmt-nix:
https://github.com/numtide/treefmt-nix/compare/f56b1934f5f8fcab8deb5d38d42fd692632b47c2...4ef3dfdbb5ddfb9e39999a2f2b0c2637277859d4

wolfgangwalther commented:
I guess so. I'd hope that upstream (treefmt-nix) can find a better fix. I haven't seen these hashes change in months, so I'm not sure why they did.

zowoq commented:
I was expecting this formatter to stay disabled until it was properly fixed in treefmt-nix.

MattSturgeon commented:
> I was expecting this formatter to stay disabled until it was properly fixed in treefmt-nix.

Let's not block re-enabling on that. If the issue reoccurs before there's a proper fix, we can always disable biome again.

I agree it'd be better to be able to disable the validation though, maybe I'll find time to send a PR.

MattSturgeon committed:
Revert "ci/treefmt: disable biome for now"

This reverts commit 66260cc8c48f38529c559ce9529d9398bc8d4837.

MattSturgeon committed:
ci/treefmt: disable biome settings validation

The treefmt-nix `biome.settings` validation uses inputs that are liable
to hash mismatch.

See https://github.com/numtide/treefmt-nix/pull/430

MattSturgeon commented on ci/pinned.json:
```diff
@@ -22,9 +22,9 @@
-      "hash": "1klcfmqb4q4vvy9kdm5i9ddl26rhlyhf1mrd5aw1d4529bqnq5b5"
+      "revision": "144d1e80d642c50c71846c0cb674e86c956e9a24",
+      "url": "https://github.com/numtide/treefmt-nix/archive/144d1e80d642c50c71846c0cb674e86c956e9a24.tar.gz",
+      "hash": "0c11v8v9mk6vzx8l0gwrbgf85gws5xhj4x108gyjbc0hikqnaz48"
```
Done

philiptaron commented on doc/tests/manpage-urls.nix:
```diff
@@ -2,12 +2,12 @@
 }:
 
-invalidateFetcherByDrvHash (
+testers.invalidateFetcherByDrvHash (
```
Excellent!

wolfgangwalther commented:
Thanks for updating the PR, @MattSturgeon. I just adjusted the commit message with the new commit hash for treefmt-nix as well.
